packages/livecd/grub2/build.yaml

```diff
@@ -1,28 +1,35 @@
-image: opensuse/leap:15.3
+image: {{ .Values.image }}
```

Comment:
If we are switching image here, the `zypper` call below should be adjusted to take into account blue and orange

Reply:
indeed, Im wondering if it makes sense to have this package in other flavors....

Reply:
Previously we were using files from opensuse for the other flavors too, we did that to let the user just specify one repo for flavor.. maybe we can do the same on aarch64 too?